cmd/deployment-check/deployment.go

```diff
@@ -258,25 +298,81 @@ func createDeployment(ctx context.Context, deploymentConfig *v1.Deployment) chan
 	return createChan
 }
 
+// multipleToelrations splits multiple toleration values entered from spec file as a string
+func multipleTolerations(splitEnvVars []string) ([]corev1.Toleration, error) {
+
+	if splitEnvVars == nil {
+		err := errors.New("nil was entered into multiple toleration parser")
+		ReportFailureAndExit(err)
+	}
```

Comment:
you do a check in the tolerationHandler func that makes sure the list has more than 1 item:
```
if len(splitEnvVars) > 1 {
```
so would this check `if splitEnvVars == nil` be necessary?

Reply:
I received feedback to always have something like this in to make sure that no one is abusing the function. I understand it is repetitive. Let me know how you want me to proceed.

Reply:
thats fair, i wasn't thinking that it was a function that could be reused! 👍